File: FrontEnd-ProjectReact/my-react-app-inventory/src/types/userAccount.ts
import type { Role } from "./role";

export interface UserAccount {
  id: number;
  username: string;
  passwordHash: string;
  fullName?: string | null;
  email?: string | null;
  roleId?: number | null;
  recStatus?: string | null;
  role?: Role | null;
}

File: Sample/Sample.Infrastructure/EntityFramework/UserAccount.cs
using System;
using System.Collections.Generic;

namespace Sample.Infrastructure.EntityFramework
{
    public partial class UserAccount
    {
        public int Id { get; set; }
        public string Username { get; set; } = null!;
        public string PasswordHash { get; set; } = null!;
        public string? FullName { get; set; }
        public string? Email { get; set; }
        public int? RoleId { get; set; }
        public string? RecStatus { get; set; }

        public virtual Role? Role { get; set; }
    }
}
